packages/desktopjs-electron/src/electron.ts

```diff
@@ -370,24 +382,28 @@ export class ElectronContainer extends WebContainerBase {
                     });
                 });
             }
-        } catch (e) {
-            console.error(e);
-        }
-
-        let replaceNotificationApi = ElectronContainer.replaceNotificationApi;
-        if (options && typeof options.replaceNotificationApi !== "undefined") {
-            replaceNotificationApi = options.replaceNotificationApi;
-        }
 
-        if (replaceNotificationApi) {
-            this.registerNotificationsApi();
-        }
+            if (options && options.autoStartOnLogin) {
+                this.electron.setLoginItemSettings({
```

Comment:
Does this have to be done in the main process or can it be done in the renderer with node enabled?

Reply:
According to this - https://www.electronjs.org/docs/api/app#appsetloginitemsettingssettings-macos-windows
it should be on the main process itself.